.github/actions/check-spelling/action.yml

```diff
@@ -0,0 +1,28 @@
+name: Check Spelling
+description: Checks for spelling errors in files.
+
+inputs:
+  example-input:
+    type: string
+    description: 'Debug'
+    required: false
+permissions:
+  contents: read
+  pull-requests: read
+  actions: read
+# outputs:
+#   followup: ${{ steps.spelling.outputs.followup }}
+runs-on: ubuntu-latest
+
+runs:
+  using: "composite"
+  steps:
+  - name: check-spelling
+    id: spelling
+    uses: check-spelling/check-spelling@prerelease
```

Comment:
https://lombiq.atlassian.net/wiki/spaces/DEV/pages/2545614855/Writing+GitHub+Actions+workflows+and+actions#:~:text=Always%20pin%20the%20action%20to%20a%20specific%20full%2Dlength%20commit%20SHA%2C%20not%20a%20version%20tag%2C%20since%20this%20ensures%20that%20we%20security%20use%20a%20given%20version.%20For%20readability%2C%20also%20comment%20the%20version%20there.

Reply:
This isn't addressed.